src/main/java/lotto/domain/LottoStore.java

```diff
@@ -5,19 +5,25 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
-
-import static lotto.domain.constant.LottoConstants.LOTTO_SIZE;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class LottoStore {
 
     private final List<Lotto> lottos;
+    private int manualLottoCount;
 
-    private LottoStore(List<Lotto> lottos){
-        validateLottos(lottos);
-        this.lottos = lottos;
+    private LottoStore(List<Lotto> autoLottos,List<Lotto> manualLotts ){
```

Comment:
앗 manualLottos 오타가 있네요
```suggestion
    private LottoStore(List<Lotto> autoLottos,List<Lotto> manualLottos ){
```